ipaserver/plugins/dns.py

```diff
@@ -2783,6 +2785,10 @@ def pre_callback(self, ldap, dn, entry_attrs, attrs_list, *keys, **options):
         if options.get('force'):
             options['skip_nameserver_check'] = True
 
+        if 'idnssoaserial' in options:
+            logger.warning("'idnssoaserial' option is deprecated."
+                           "Value will be ignored.")
+
```

Comment:
This will log on the server side. I guess it's a heads-up of sorts.  You want to look at messages.add_message() instead.